aikau/src/main/resources/alfresco/layout/FixedHeaderFooter.js

```diff
@@ -225,12 +240,33 @@ define(["alfresco/core/ProcessWidgets",
       },
 
       /**
+       * Extends the [inherited function]{@link module:alfresco/core/CoreWidgetProcessing#allWidgetsProcessed}
+       * to set up subscriptions for the [visibilityRuleTopics]{@link module:alfresco/layout/DynamicVisibilityResizingMixin#visibilityRuleTopics}
+       * that are returned by calling [getVisibilityRuleTopics]{@link module:alfresco/layout/DynamicVisibilityResizingMixin#getVisibilityRuleTopics}.
+       * The subscriptions need to be created after the widgets have been created in order that their visibility 
+       * is adjusted before the [onResize]{@link module:alfresco/layout/FixedHeaderFooter#onResize} function that is bound 
+       * to is called.
+       *
+       * @instance
+       * @param {object[]} widgets The widgets that have been created
+       * @since 1.0.38
+       */
+      allWidgetsProcessed: function alfresco_layout_HorizontalWidgets__allWidgetsProcessed(/*jshint unused:false*/ widgets) {
+         this._allWidgetsProcessedCount++;
+         if (this._allWidgetsProcessedCount === 3)
```

Comment:
3 seems a bit arbitrary ... could this rust easily as the code around it changes?